src/components/molecules/NavBar/NavBar.scss

```diff
@@ -150,55 +159,103 @@ $border-radius: 28px;
     }
   }
 }
-.nav-party-logo {
-  position: relative;
-  background-color: rgba($white, 0.08);
-  height: 60px;
-  padding: 0 50px 0 20px;
-  text-align: center;
-  cursor: pointer;
+
+.nav-schedule {
+  position: fixed;
   display: flex;
-  justify-content: space-between;
+  flex-direction: column;
   align-items: center;
-  overflow: hidden;
-  white-space: nowrap;
-  text-overflow: ellipsis;
-  transition: transform 400ms $transition-function;
-  border: none;
-  appearance: none;
+  justify-content: center;
+  top: 70px;
+  left: 0px;
+  background-color: $primary;
   color: $white;
+  width: $nav-container-width;
+  height: $nav-container-height;
+  border-top-right-radius: 10px;
+  border-bottom-right-radius: 10px;
+  cursor: pointer;
 
-  img {
-    height: 40px;
+  &-clicked {
+    margin-left: 40%;
+    background-color: $schedule-opened-grey;
+    width: 30px;
   }
-  &:hover,
-  &:focus {
-    border: none;
-    background-color: rgba($white, 0.14);
-    &:after {
-      opacity: 0.7;
+
+  &-triggered {
+    background-color: $lighter-intermediate-grey;
+  }
+
+  &__text {
+    display: flex;
+    width: $nav-text-body-width;
+    height: $nav-text-height;
+    flex-direction: column;
+    align-items: center;
+
+    &-first {
+      width: $nav-text-head-width;
+      height: $nav-text-body-height;
+      font-size: 20px;
+      background-image: url(/assets/icons/nav-schedule-whats.png);
+    }
+    &-second {
+      width: $nav-text-body-width;
+      height: $nav-text-body-height;
+      font-size: 22px;
+      background-image: url(/assets/icons/nav-schedule-on.png);
```

Comment:
why haven't you used 1 icon instead of 3? 

https://sapiens.chat/sparkle/styleguide/assets/styleguide/schedule-sidebar-icon.png

Reply:
using 2 icons atm, 1 for normal state and 1 for arrow only
e1ea69a